xml/System.Security.Cryptography.X509Certificates/X509Certificate.xml

```diff
@@ -2656,6 +2671,7 @@
         <permission cref="T:System.Security.SecurityCriticalAttribute">requires full trust for the immediate caller. This class cannot be used by partially trusted or transparent code.</permission>
         <permission cref="F:System.Security.Permissions.SecurityAction.InheritanceDemand">for full trust for inheritors. This member cannot be inherited by partially trusted code.</permission>
         <permission cref="T:System.Security.Permissions.KeyContainerPermission">for permission to create a key container. Security action: <see cref="F:System.Security.Permissions.SecurityAction.Demand" />. Associated enumeration: <see cref="F:System.Security.Permissions.KeyContainerPermissionFlags.Create" /></permission>
+        <exception cref="T:System.PlatformNotSupportedException">.NET Core only: In all cases.</exception>  
```

Comment:
Should we add a remark to all these APIs indicating the reason why we throw `PlatformNotSupportedException`?